### PR TITLE
[Spotless] Applying Google Code Format for core/src/main files #3

### PR DESCRIPTION
**Pull request by MitchellGale**
### Description

This PR applies Spotless on Java code for:
`core/src/main/java/org/opensearch/sql/planner`
`core/src/main/java/org/opensearch/sql/storage`
`core/src/main/java/org/opensearch/sql/utils` 

 
### Issues Resolved
https://github.com/opensearch-project/sql/issues/1101
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [x] New functionality has javadoc added
  - [x] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: core/src/main/java/org/opensearch/sql/planner/physical/RemoveOperator.java [hunk 1]
```diff
@@ -24,26 +23,19 @@
...
-  @Getter
-  private final PhysicalPlan input;
-  @Getter
-  private final Set<ReferenceExpression> removeList;
-  @ToString.Exclude
-  @EqualsAndHashCode.Exclude
-  private final Set<String> nameRemoveList;
+  @Getter private final PhysicalPlan input;
+  @Getter private final Set<ReferenceExpression> removeList;
+  @ToString.Exclude @EqualsAndHashCode.Exclude private final Set<String> nameRemoveList;
```
Comment (GumpacG) [inline core/src/main/java/org/opensearch/sql/planner/physical/RemoveOperator.java:32]: How come the multiple annotations are put on one line here while multiple annotations for the function above are left on multiple lines.

**Review by GumpacG — APPROVED** [review]

File: core/src/main/java/org/opensearch/sql/planner/physical/RemoveOperator.java (hunk 1 above)
Comment (acarbonetto) [reply core/src/main/java/org/opensearch/sql/planner/physical/RemoveOperator.java:32]: You'll have to look at the rules, but I think the idea is that classes aren't indented so the annotations are put above, while functions and class variables are already indented, so it doesn't hurt to indent them more...?

**Review by acarbonetto — DISMISSED** [review]

**Review by Yury-Fridlyand — APPROVED** [review]
Please also fix the PR description

File: core/src/main/java/org/opensearch/sql/planner/optimizer/rule/PushFilterUnderSort.java [hunk 2]
```diff
@@ -19,10 +18,7 @@
 import org.opensearch.sql.planner.logical.LogicalSort;
 import org.opensearch.sql.planner.optimizer.Rule;
 
-/**
- * Push Filter under Sort.
- * Filter - Sort - Child --> Sort - Filter - Child
- */
+/** Push Filter under Sort. Filter - Sort - Child --> Sort - Filter - Child */
```
[the remark this answers is not in the document]
Comment (MitchellGale) [reply core/src/main/java/org/opensearch/sql/planner/optimizer/rule/PushFilterUnderSort.java:21]: Addressed.

File: core/src/main/java/org/opensearch/sql/planner/physical/NestedOperator.java [hunk 3]
```diff
@@ -128,25 +117,27 @@ public ExprValue next() {
...
         this.nonNestedFields.add(inputField.getKey());
       }
     }
   }
 
-
   /**
+   *
+   *
+   * <pre>
```
[the remark this answers is not in the document]
Comment (MitchellGale) [reply core/src/main/java/org/opensearch/sql/planner/physical/NestedOperator.java:140]: Addressed.

**Review by GumpacG — APPROVED** [review]

**Review by Yury-Fridlyand — APPROVED** [review]

**Review by GumpacG — APPROVED** [review]

**Review by Yury-Fridlyand — APPROVED** [review]

**Pull request merged**